app/controllers/gym_memberships_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GymMembershipsController < ApplicationController
```

Comment:
Placeholder - but we need to move `GymsController.destroy` here, and fix the routes.  I would recommend we clean this up proactively as a part of this PR